.gitignore

```diff
@@ -7,3 +7,5 @@ node_modules
 /skeleton/package-lock.json
 /skeleton/public/app.js
 /skeleton/public/main.css
+
+.DS_Store
```

Comment:
I would recommand doing it in your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) as `.DS_Store` files are specific to mac users.

Reply:
Came here to say the exact same

PS: IT'S SATURDAY FOLKS

Reply:
Yes i know that it's saturday! But you could reviewed on monday ;) not today!